server/handlers/utils_test.go

```diff
@@ -0,0 +1,17 @@
+package handlers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/theupdateframework/notary/tuf/data"
+	"github.com/theupdateframework/notary/tuf/signed"
+	"github.com/theupdateframework/notary/tuf/testutils"
+)
+
+func mustCopyKeys(t *testing.T, from signed.CryptoService, roles ...data.RoleName) signed.CryptoService {
+	cs, err := mustCopyKeys(t, from, roles...)
```

Comment:
shouldnt that be `testutils.CopyKeys`?

Reply:
Ah, dang; you're right. Must've done find/replace _after_ I added that file 🤦‍♂️ and forgot to update. Fixing now